dbt/adapters/databricks/python_submissions.py

```diff
@@ -1,14 +1,315 @@
-from typing import Dict, Optional
-
-from dbt.adapters.spark.python_submissions import (
-    AllPurposeClusterPythonJobHelper,
-    BaseDatabricksHelper,
-    JobClusterPythonJobHelper,
-)
+from typing import Any, Dict, Tuple, Optional, Callable
 
 from dbt.adapters.databricks.__version__ import version
 from dbt.adapters.databricks.connections import DatabricksCredentials
 
+import base64
+import time
+import requests
+import uuid
+
+import dbt.exceptions
+from dbt.adapters.base import PythonJobHelper
+from dbt.adapters.spark import SparkCredentials
```

Comment:
why do we need these dependencies?

Reply:
If there is no drawback, I'd rather decouple from dbt-spark where ever we can. 
Also, do we need to port tests? Or is this code tested on our side?

Reply:
Thanks @andrefurlan-db. Done! It was actually easier than I thought because we already made a `DatabricksCredentials` class -- I just needed to update the type hints and remove the unnecessary `SparkCredentials` import. All tests pass.

As for the tests: we already duplicated the dbt-spark tests on our side in the past so that work is done 🙌